sdk/v2/data.go

```diff
@@ -196,12 +195,12 @@ func (integration *Integration) toJSON(pretty bool) (string, error) {
 	}
 
 	if err != nil {
-		return "", fmt.Errorf("error marshalling to JSON: %s", err)
+		return []byte(""), errors.Errorf("error marshalling to JSON: %s", err)
```

Comment:
Do not create a new slice of byte but return `output` var directly

Reply:
Why? That'd change the current behaviour